sources/api/migration/migrations/v1.19.2/certdog-config-file-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use migration_helpers::common_migrations::AddPrefixesMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// Add settings for the new certdog-toml config file
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "services.pki",
```

Comment:
This isn't correct - `services.pki` was added a long time ago (in v1.2.0).

`AddPrefixesMigration` could also be written as `RemovePrefixesOnDowngradeMigration` - it's a no-op on upgrade but does the equivalent of `rm -rf` on downgrade. If used incorrectly then previously valid settings may go missing.